Gems/MotionMatching/Code/motionmatching_tests_files.cmake

```diff
@@ -10,5 +10,6 @@ set(FILES
     Tests/Fixture.h
     Tests/FeatureMatrixTests.cpp
     Tests/FeatureSchemaTests.cpp
+    Tests/MinMaxScalerTests.cpp
     Tests/MotionMatchingTest.cpp
 )
```

Comment:
uber nit: Could add eof here while we're here 🙂

Reply:
Can you elaborate what you mean with that? ;) I am not aware of a special manual eof character that we should use in cmake files?

Reply:
I think it just means the file needs to end with a newline character. I'm not sure what the reasoning behind that is - I can't find it in the coding standards.